llama-demo-apps/BreakingNews.ipynb

```diff
@@ -0,0 +1,406 @@
+      "        \"A new battle over abortion in the U.S. is threatening the President\\u2019s Emergency Plan for AIDS Relief, PEPFAR, which provides HIV/AIDS medications to millions around the world including at an orphanage in Nairobi, Kenya. (Sept.11) (AP video: Josphat Kasire & Dan Huff)\",\n",
+      "        \"Dictionary.com Editorial Vice President John Kelly says pop culture and social media have a strong influence on the new words and phrases \\u2013 and the changes to existing definitions\\u2014in the site\\u2019s latest update. (Sept. 7)\"\n",
+      "      ],\n",
+      "      \"title\": \"Associated Press News: Breaking News, Latest Headlines and Videos ...\",\n",
+      "      \"url\": \"https://apnews.com\"\n",
+      "    },\n",
+      "    {\n",
+      "      \"description\": \"Chicago <strong>breaking</strong> <strong>news</strong>, weather and live video. Covering local politics, health, traffic and sports for Chicago, the suburbs and northwest Indiana.\",\n",
+      "      \"snippets\": [\n",
+      "        \"Johnson planning border trip; Migrants dropped off at Willis Tower\\nMayor Brandon Johnson said Wednesday that he will be making a trip to the border to see the migrant crisis first-hand.\\nShow More\\nTop Videos\\nABC7 Chicago 24/7 Stream\\nJohnson planning border trip; Migrants dropped off at Willis Tower\\nAttorneys for ex-NU coach Fitzgerald expected to announce lawsuit\\nU.S. & World\\nShow More\\nMore Accuweather\\nMore Shop\\nFeatured\\nShow More\\nShow Fewer\\nMore Promotions\"\n",
+      "      ],\n",
+      "      \"title\": \"ABC Chicago - Breaking News and Weather\",\n",
+      "      \"url\": \"https://abc7chicago.com/\"\n",
+      "    }\n",
+      "  ],\n",
+      "  \"latency\": 0.7186734676361084\n",
+      "}\n"
+     ]
+    }
+   ],
+   "source": [
+    "# check the query result in JSON\n",
+    "import json\n",
+    "\n",
+    "print(json.dumps(data, indent=2))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "7c40e73f-ca13-4f4a-a753-e613df3d389e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# one way to load the JSON result with key info stored as \"snippets\"\n",
+    "from llama_index import download_loader\n",
+    "\n",
+    "JsonDataReader = download_loader(\"JsonDataReader\")\n",
+    "loader = JsonDataReader()\n",
+    "documents = loader.load_data([hit[\"snippets\"] for hit in data[\"hits\"]])"
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
feel this cell need to go before the previous cell. one nit: I think adding actual md/ text cells help to look more like a tutorial.

Reply:
i merged the cell to the previous one to be more consistent with other cells and the style of the whole notebooks, as i played with changing it to markdown cell and it looks too different.